frontend/src/pages/collab.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import { io } from 'socket.io-client';
+
+const socket = io('http://localhost:5001/test');
```

Comment:
I think port 5001 is for user-service while 5003 should be for the collab service

Reply:
Ah I see thanks